src/widget/form/groupinviteform.cpp

```diff
@@ -149,8 +153,29 @@ void GroupInviteForm::onGroupInviteRejected()
     int index = inviteLayout->indexOf(groupWidget);
     groupInvites.removeAt(index);
 
-    groupWidget->deleteLater();
-    inviteLayout->removeWidget(groupWidget);
+    deleteInviteButtons(groupWidget);
+}
+
+void GroupInviteForm::deleteInviteButtons(QWidget *widget)
+{
+    QList<QPushButton*> buttons = widget->findChildren<QPushButton*>();
+
+    if (acceptButtons.contains(buttons.at(0)))
```

Comment:
We have structure like that:
all requests -> request -> buttons
And "button->parentWidget()" return "request", which contains only 2 buttons